edb/server/protocol/auth_ext/http.py

```diff
@@ -2287,7 +2315,7 @@ def _fail_with_error(
         "type": str(ex.__class__.__name__),
     }
 
-    logger.error(f"Failed to handle HTTP request: {err_dct!r}")
+    logger.error(f"Failed to handle HTTP request: {err_dct!r}", exc_info=True)
```

Comment:
Would this be too noisy? As `_fail_with_error()` is used by many handlers, maybe it's better if we:

```
def _fail_with_error(..., exc_info: bool):
    ....
    logger.error(...., exc_info=exc_info)
```

and call `_fail_with_error(exc_info=True)` only in branches that - we know - are handling unexpected errors, like `except Exception` or arguably `except WebAuthnRegistrationFailed`

Reply:
Fair question. One of the goals of 6.x for the auth extension was to make it easier to troubleshoot and debug while developing, so I'd personally rather show more and more helpful errors in more places than continue to be conservative here, but maybe this is too large a change for too surgical of a bug fix to warrant making big moves here. I'm fine with your suggestion of reducing the scope of this change and re-addressing the question of logging in it's own context.

Reply:
@fantix ⚡ [`5e02c24` (#8428)](https://github.com/geldata/gel/pull/8428/commits/5e02c2453010fa1cb16e55e8cfd37158dcc5bc2f)